citadel.dme

```diff
@@ -5683,6 +5676,14 @@
 #include "maps\shuttles\factions\corporations\nanotrasen\drone_prototype.dm"
 #include "maps\shuttles\factions\corporations\nanotrasen\sci_vector.dm"
 #include "maps\shuttles\factions\generic\outlaw\reaver.dm"
+#include "maps\strelka\strelka-areas.dm"
+#include "maps\strelka\strelka-misc.dm"
+#include "maps\strelka\strelka-overmap.dm"
+#include "maps\strelka\strelka-papers.dm"
+#include "maps\strelka\strelka-sectors.dm"
+#include "maps\strelka\strelka-shuttles.dm"
+#include "maps\strelka\strelka.dm"
+#include "maps\strelka\telecomms.dm"
```

Comment:
You might end up with merge conflicts due to this, just yell at one of us if it happens